src/targets/browser/index.jsx

```diff
@@ -67,12 +68,17 @@ const renderApp = function(appLocale, client, isPublic) {
           <CozyProvider client={client}>
             <MuiCozyTheme>
               <IsPublicContext.Provider value={isPublic}>
-                {!isPublic && (
-                  <SharingProvider doctype="io.cozy.files" documentType="Notes">
-                    <App isPublic={isPublic} />
-                  </SharingProvider>
-                )}
-                {isPublic && <App isPublic={isPublic} />}
+                <WebviewIntentProvider>
+                  {!isPublic && (
+                    <SharingProvider
+                      doctype="io.cozy.files"
+                      documentType="Notes"
+                    >
+                      <App isPublic={isPublic} />
+                    </SharingProvider>
+                  )}
+                  {isPublic && <App isPublic={isPublic} />}
+                </WebviewIntentProvider>
```

Comment:
maybe we could test that 🙏

Reply:
I'm not sure I see value in testing this, it's too declarative. The provider itself is tested in cozy-intent